drydock_provisioner/error.py

```diff
@@ -44,9 +44,16 @@ def __init__(self, msg, code=500):
         self.status_code = code
 
     def to_json(self):
-        err_dict = {'error': msg, 'type': self.__class__.__name__}}
+        err_dict = {'error': msg, 'type': self.__class__.__name__}
         return json.dumps(err_dict)
 
 class InvalidFormat(ApiError):
     def __init__(self, msg, code=400):
         super(InvalidFormat, self).__init__(msg, code=code) 
+
+class ClientError(Exception):
+    def __init__(self, msg, code=500):
```

Comment:
Client errors are 400-related codes. See section 10.4 of https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html